README.md

```diff
@@ -1,7 +1,8 @@
 # Probabilistic Numerics in Python
 
-[![Build Status](https://travis-ci.org/probabilistic-numerics/probnum.svg?branch=master)](https://travis-ci.org/probabilistic-numerics/probnum)
+[![Build Status](https://img.shields.io/travis/probabilistic-numerics/probnum/master?logo=travis%20ci&logoColor=white&label=Travis%20CI)](https://travis-ci.org/probabilistic-numerics/probnum)
```

Comment:
As I understood it, the banner from travis-ci.org shows the actual build status. Personally I would prefer that and consider it to be a bit more informative than an "always green" banner

Reply:
This should still be the behaviour. Compare also: https://github.com/astropy/astropy/blob/master/README.rst

Reply:
My bad, I completely misjudged the capabilities of shields.io!